neqo-transport/src/recv_stream.rs

```diff
@@ -557,12 +509,42 @@ impl RecvStream {
             }
         }
     }
+
+    /// Maybe write a `MAX_STREAM_DATA` frame.
+    pub fn write_frame(
+        &mut self,
+        builder: &mut PacketBuilder,
+        tokens: &mut Vec<RecoveryToken>,
+        stats: &mut FrameStats,
+    ) -> Res<()> {
+        // Send MAX_STREAM_DATA at normal priority always.
```

Comment:
```suggestion
```
No point in mentioning this.  (We could prioritize receive, but it doesn't seem like it would help that much.)

Reply:
yes, I decided to leave it at normal priority.